packages/vite/src/node/server/hmr.ts

```diff
@@ -461,15 +461,15 @@ async function restartServer(server: ViteDevServer) {
   // @ts-ignore
   global.__vite_start_time = Date.now()
   const { port } = server.config.server
-  
+
```

Comment:
I think we should revert the changes to this file to avoid having it in the history diff

Reply:
Good catch!